src/mds/Server.cc

```diff
@@ -1720,7 +1720,9 @@ void Server::handle_osd_map()
    * using osdmap_full_flag(), because we want to know "is the flag set"
    * rather than "does the flag apply to us?" */
   mds->objecter->with_osdmap([this](const OSDMap& o) {
-      is_full = o.test_flag(CEPH_OSDMAP_FULL);
```

Comment:
auto pi = o.get_pg_pool(mds->mdsmap->get_metadata_pool());
is_full = pi && pi->has_flag(pg_pool_t::FLAG_FULL);

Reply:
I was looking for that and couldn't find the right thing. Thanks!

Reply:
Fixed.